backend/app/routes/events.py

```
@@ -438,3 +408,183 @@ def get_croatian_event_recommendations(
         ],
         "cultural_categories": cultural_categories,
     }
+
+
+@router.post("/geocode")
+async def geocode_events(
+    limit: int = Query(50, ge=1, le=200, description="Number of events to geocode"),
+    db: Session = Depends(get_db),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)